frontend/src/embedding-sdk-bundle/components/private/SdkQuestionDefaultView/SdkQuestionDefaultView.tsx

```diff
@@ -53,7 +53,7 @@ import InteractiveQuestionS from "./SdkQuestionDefaultView.module.css";
 
 export interface SdkQuestionDefaultViewProps extends FlexibleSizeProps {
   /**
-   * Determines whether the question title is displayed, and allows a custom title to be displayed instead of the default question title. Shown by default. Only applicable to interactive questions when using the default layout.
+   * Determines whether the question title is displayed, and allows a custom title to be displayed instead of the default question title. Shown by default.
```

Comment:
We probably want to remove the mentioning of interactive question from `withDownloads` too. Since it seems to work with `StaticQuestion` as well.

<img width="749" height="129" alt="Image" src="https://github.com/user-attachments/assets/f941cc3f-e61a-43d0-b5ef-ae121fe38d7b" />